scripts/example_script.py

```diff
@@ -32,5 +37,14 @@
 create_mailchimp_connector(
     auth_header=auth_header,
 )
+create_stripe_connector(
+    auth_header=auth_header,
+)
+
+create_mongodb_connector(auth_header=auth_header)
+configure_s3_storage(auth_header=auth_header)
+
+create_policy(auth_header=auth_header)
```

Comment:
I think the policy creation needs to come before s3 storage

Reply:
It did originally. I updated the `configure_s3_storage` function to call the `create_policy` function from within it. I ran into an issue where it failed to create the config because a policy wasn't created yet. Since the `create_policy` function calls a patch route it won't create the extra policies. Instead, it will update the original if it already exists. 

If the preferred method is calling the two functions separately I can revert the `configure_s3_storage` to not call `create_policy` from within.